engine/apps/public_api/serializers/schedules_calendar.py

```diff
@@ -11,6 +11,7 @@
 
 
 class ScheduleCalendarSerializer(ScheduleBaseSerializer):
+    team_id = TeamPrimaryKeyRelatedField(required=False, allow_null=True, source="team")
```

Comment:
There is team_id field in  ScheduleBaseSerializer, is second one necessary?

Reply:
```suggestion
```